### PR TITLE
Change `tootctl media remove` flags for clarity

### PR DESCRIPTION
**Pull request by evanphilip**
Change `tootctl media remove` flags for clarity https://github.com/mastodon/mastodon/issues/23628

# changelog
* Add `tootctl media remove` flags `--attachments`, `--avatars` and `--headers`.
* Deprecate  `tootctl media remove` flags `--prune-profiles` and `--remove-headers`.
* Deprecate use of  `tootctl media remove` without flags specifying media.
* Fix throwing of error on use of `--verbose` flag with `tootctl media remove` https://github.com/mastodon/mastodon/issues/24070

### CODE REVIEW

File: lib/mastodon/media_cli.rb [hunk 1]
```diff
@@ -14,52 +14,71 @@ def self.exit_on_failure?
...
+    For backward compatibility, media attachments will be removed if no flags specifying media are provided, but this is deprecated. In future releases, if no flags are provided, nothing will be removed.
+    --attachments
+        Include media attachments for removal.
+    --avatars
+        Include account avatars for removal.
+    --headers
+        Include profile headers for removal.
+    --include-follows
+        Remove media even if a follow relationship exists.
+        If this flag is not provided, only accounts that are not followed by or following anyone locally will have their media removed.
```
Comment (trwnh) [inline lib/mastodon/media_cli.rb:40]: long lines should be wrapped

**Comment by evanphilip** [comment]
- [ ] What is the preferred way to mark deprecation in code?
- [ ] Does `Follow.where(account: media_attachment.account)` work?

**Comment by mjankowski** [comment]
This needs a rebase (which might also solve some of the annotation issues currently in the diff?) ... and might I suggest a smaller series of PRs each doing one of these changes at a time? Might be easier to review.

**Comment by mjankowski** [comment]
I pulled out the verbose fix to https://github.com/mastodon/mastodon/pull/30536

Closing this due to lack of activity/rebase. If you want to nudge forward, can you rebase, re-open, deal with lint issues, etc.

**Pull request closed without merging**